lib/services/plugins-service.ts

```diff
@@ -49,6 +49,11 @@ export class PluginsService implements IPluginsService {
 				this.$errors.fail("No plugin name specified");
 			}
 
+			var pluginNameToLowerCase = pluginName.toLowerCase();
+			if(!_.any(this.getAvailablePlugins(), (pl) => pl.data.Name.toLowerCase() ===  pluginNameToLowerCase || pl.data.Identifier.toLowerCase() === pluginNameToLowerCase)) {
+				this.$errors.fail("Invalid plugin name: %s", pluginName);
```

Comment:
`failWithoutHelp` seems more appropriate for me, it definitely looks like the user knows how to use the command, just the plugin id is incorrect.